test/acceptance/behave/README.md

```diff
@@ -183,13 +183,13 @@ The log is stored in `logs` folder and is called `behave.log` see `logging.ini`.
     * list_an_entity_by_id - 209 testcases              GET - /v2/entities/<entity_id>
     * list_an_attribute_by_id - 198 testcases           GET - /v2/entities/<entity_id>/attrs/<attr_name>
     * update_append_attribute_by_id - 766 testcases     POST - /v2/entities/<entity_id> plus payload
-    * update_attribute_by_id - 654 testcases            PATCH - /v2/entities/<entity_id> plus payload  
-    * replace_attributes_by_id (pending)
-    * remove_entity (pending)
+    * update_attribute_by_id - 654 testcases            PATCH - /v2/entities/<entity_id> plus payload
+    * replace_attributes_by_id 511 testcases            PUT - /v2/entities/<entity_id> plus payload  
+    * remove_entity (pending) 
 
   -  alarms (pending)
-
-
+  
+  
 ### Hints:
```

Comment:
It is something of windows to linux (replacing CRLF) , first is removed two lines and after is added two lines.